src/libcrun/cgroup.c

```diff
@@ -2372,7 +2372,7 @@ write_cpu_resources (int dirfd_cpu, bool cgroup2, runtime_spec_schema_config_lin
       if (UNLIKELY (ret < 0))
         return ret;
     }
-  if (cpu->realtime_runtime_present)
+  if (cpu->realtime_runtime)
```

Comment:
Should these be || statements?

Reply:
`realtime_runtime` can be `!= 0` only when its value is set.  The `_present` variant is useful only to check in case its value is 0, whether it was set by the config file or it is just unset.

What the change does is to ignore both cases when it is unset and when its value is `0`.